tests/ranger-client.bats

```diff
@@ -5,14 +5,25 @@ RANGER_PORT=6080
 RANGER_REPO=hivedev
 RANGER_URL=http://${RANGER_HOST}:${RANGER_PORT}
 HIVE_LIB=/usr/lib/hive1.2
-RANGER_HIVE_PLUGIN_PATH=/media/ephemeral0/hive_plugin/ranger-1.1.0-hive-plugin
+PLUGIN_VER=1.1.0
+RANGER_HIVE_PLUGIN_PATH=/media/ephemeral0/hive_plugin/ranger-${PLUGIN_VER}-hive-plugin
 
 function setup() {
     if [[ ! -e /tmp/RANGER_INSTALLED ]]; then
         source /usr/lib/qubole/bootstrap-functions/hive/ranger-client.sh
         install_ranger -h ${RANGER_HOST} -p ${RANGER_PORT} -r ${RANGER_REPO}
         touch /tmp/RANGER_INSTALLED
     fi
+    source /usr/lib/qubole/bootstrap-functions/common/utils.sh
+    HIVE_VERSION=$(nodeinfo hive_version)
+    if [ "${HIVE_VERSION}" = "2.3" ]; then
```

Comment:
It would be better if we can maintain a key value pairs of hive_version and ranger_plugin_version (if it is possible with OS or shell we use) so that it would be easy to make changes when a new version is introduced in future.